### ISSUE TITLE
PEAR.ControlStructures.MultiLineCondition.Alignment not detected if closing brace is first token on line

### ISSUE THREAD

**Issue by nkovacs** [squizlabs/PHP_CodeSniffer#1138]
PEAR.ControlStructures.MultiLineCondition.Alignment is detected here for the closing brace:

```
<?php

function bar() {
    if ($a
        && $b
 ) {
        echo 'bar';
    }
}
```

But not here (because the loop exits before the closing brace):

```
<?php

function bar() {
    if ($a
        && $b
) {
        echo 'bar';
    }
}
```

Unfortunately, the autofixer produces code like the second one when fixing the CloseBracket error here:

```
<?php

function bar() {
    if ($a
        && $b) {
        echo 'bar';
    }
}
```

(try this with `phpcbf --standard=PEAR --exclude=PEAR.WhiteSpace.ScopeIndent`)

The ScopeIndent Sniff fixes the second example if it is enabled, but I think this should be detected and fixed by the MultiLineCondition sniff as well (since it does so if there's at least one space).

**Comment by gsherwood** [comment]
I've added a fix for this. Thanks for the bug report.
